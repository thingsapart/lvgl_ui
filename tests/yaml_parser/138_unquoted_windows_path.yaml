# Backslashes in unquoted strings are literal characters.
# cJSON output will escape them.
path: C:\Users\Default\Documents
