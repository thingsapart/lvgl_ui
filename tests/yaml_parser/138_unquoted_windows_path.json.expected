[{
		"path":	"C:\\Users\\Default\\Documents"
	}]
